packages/compass-e2e-tests/helpers/insert-data.ts

```diff
@@ -66,6 +66,7 @@ export async function createDummyCollections(): Promise<void> {
   promises.push(createBlankCollection(db, 'extended-json-file'));
   promises.push(createBlankCollection(db, 'csv-file'));
   promises.push(createBlankCollection(db, 'bom-csv-file'));
+  promises.push(createBlankCollection(db, 'latin1'));
```

Comment:
Maybe not for this pr, but we're running these on every `beforeEach`. That will recreate these collections even when they aren't being used. I'm thinking we could refactor it to either run in `before` if each collection is used individually, or we could have it so a scoped `beforeEach` calls certain collections to be made so we don't have to do all of them. Would that make sense for you?

Reply:
Makes sense. We export some more specific helpers to create some collections already. I was just trying to avoid having to also make a another connection to the db inside the tests and also making sure that collections happen in known databases that we know to drop before tests run so we can guarantee that things get cleaned up.

btw, I don't think it takes much time to create an empty collection once you already have a connection open. But I don't mind changing this code either.